src/protos/flywheels.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+
+/*************************************************/
+/******** Main messages that will be sent ********/
+/*************************************************/
+
+message FlywheelConfig {
+    repeated int32 controlPins = 1;
+    telemMethod telemetry = 2;
+    // Both no telemetry and bidirectional Dshot will not have any telemetry pins
+    // Serial and tachometer will likely only have one pin, but we could support having them communicate on multiple pins
+    repeated int32 telemPins = 3;
+
+    int32 motorKv = 4;
+    dshotModes dshotMode = 5;
+}
+
+message ClosedLoopFlywheelParams {
+    int32 revRpm = 1;
+    int32 idleRpm = 2;
+    int32 idleTime_ms = 4;
+
+    // Should this be rpm? Percent of rev rpm?
+    int32 firingThresholdRpm = 5;
+
+    // Can we get some sort of unit on this?
+    int32 spindownSpeed = 6;
+}
+
+message OpenLoopFlywheelParams {
+    int32 revDutyCycle = 1;
+    int32 idleDutyCycle = 2;
+    int32 idleTime_ms = 3;
+
+    // The minimum amount of time that the flywheels must have revved before the pusher can be triggered
+    int32 minimumRevTime_ms = 4;
+    // The minimum amount of time that allows the flywheels to spin back up after firing a dart
+    int32 delayBetweenDarts_ms = 5;
```

Comment:
My thought was that this would set a floor that the pusher module can't go below because this is a constraint that the flywheels impose on the pusher, hence why it's here

Reply:
Yeah I don't think the flywheels are a limiting factor to rof with any modern nerf motor, and certainly not with brushless

Reply:
I just planned to write that floor value to the pusher module at config time, then use that to check before we allow a rate of fire profile. The web app could also check that before it sends anything.

But we can nix it if you don't think it's an issue

Reply:
Yeah thanks, I don't think it's necessary

Reply:
I'm curious about that, but I think we can resolve this conversation for now